### ISSUE TITLE
Please add APK to releases

### ISSUE THREAD

**Issue by rancidfrog** [PureWriter/PureWriter#35]
Please use release section to host apk, as an alternative to play store, as not all users have goggles services.
Thank you

**Comment by benji88i** [comment]
Yes I second that, I am having trouble restoring my pro version. I installed from a third party repo but cannot restore my purchase for the app. Did you install with pro version?
